x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/manage_processor_form/processors/common_fields/processor_type_field.tsx

```diff
@@ -68,13 +68,15 @@ export const ProcessorTypeField: FunctionComponent<Props> = ({ initialType }) =>
     <UseField<string> config={typeConfig} defaultValue={initialType} path="type">
       {(typeField) => {
         let selectedOptions: ProcessorTypeAndLabel[];
+        let helpText: string = '';
```

Comment:
It looks like we still need to assign this value to the EuiFormRow below

Reply:
You can also drop the `: string` annotation. If you init this value to a string TS will figure out that it is a string automatically.